classes/Domain/Services/ResetEmailer.php

```diff
@@ -82,8 +82,8 @@ private function parameters($userId, $resetCode)
                 ? 'https' : 'http',
             'host'    => !empty($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : 'localhost',
```

Comment:
`phpmd` complains about use of `$_SERVER`. We *could* inject an instance of `Request` here but... I don't know that I care. What do you think? And if we don't care, do we want to create an issue to adjust `phpmd` to slack-off?

Reply:
How about in a separate PR?

Reply:
The silex Application object has the ```requireHttps()``` function we could use, and the host can be retrieved from the config file ( i think). So no need to use the ```Request``` object.

Reply:
Sounds like a good plan to me, @BackEndTea!